third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h

```diff
@@ -6,6 +6,7 @@
 #define DOMWindowWebCL_h
 
 #include "core/frame/DOMWindowProperty.h"
+#include "core/frame/LocalDOMWindow.h"
```

Comment:
The declaration of `m_window` below is strange. Why it's a reference? This needs to be double checked to determine whether we should use class forward declaration or #include.